app/assets/javascripts/discourse/models/group.js.es6

```diff
@@ -1,31 +1,31 @@
+import EmberObject from "@ember/object";
+import { equal } from "@ember/object/computed";
 import { isEmpty } from "@ember/utils";
-import { notEmpty, equal } from "@ember/object/computed";
-import { ajax } from "discourse/lib/ajax";
 import {
   default as discourseComputed,
   observes
 } from "discourse-common/utils/decorators";
+import { ajax } from "discourse/lib/ajax";
+import Category from "discourse/models/category";
 import GroupHistory from "discourse/models/group-history";
 import RestModel from "discourse/models/rest";
-import Category from "discourse/models/category";
-import User from "discourse/models/user";
 import Topic from "discourse/models/topic";
-import { popupAjaxError } from "discourse/lib/ajax-error";
-import EmberObject from "@ember/object";
+import User from "discourse/models/user";
 
 const Group = RestModel.extend({
-  limit: 50,
-  offset: 0,
   user_count: 0,
+  limit: null,
+  offset: null,
+
+  request_count: 0,
```

Comment:
```suggestion
  requestCount: 0,
```

Reply:
This is similar to `user_count` above. We are building the model from the exact response we get from the server.